packages/migrations/package.json

```diff
@@ -8,19 +8,19 @@
     "directory": "packages/migrations"
   },
   "scripts": {
-    "build": "tsc && copyfiles -u 1 \"./src/**/*.sql\" dist",
-    "dev": "tsc --watch --preserveWatchOutput",
+    "build": "tsc && tscp",
+    "dev": "tsc --watch --preserveWatchOutput & tscp --watch",
```

Comment:
Does the backgrounded task die correctly when we kill `yarn dev`?

Reply:
It appears to!